src/main/kotlin/registration/eth/main.kt

```diff
@@ -16,13 +18,15 @@ private val logger = KLogging().logger
 fun main(args: Array<String>) {
     val registrationConfig =
         loadConfigs("eth-registration", EthRegistrationConfig::class.java, "/eth/registration.properties")
-    executeRegistration(registrationConfig)
+    val passwordConfig = loadEthPasswords("eth-notary", "/eth/ethereum_password.properties", args)
```

Comment:
I guess registration service should use its own prefix (e.g. "eth-registration").